src/15_classes.py

```diff
@@ -3,27 +3,59 @@
 
 # YOUR CODE HERE
 
+
+class Latlon():
+    def __init__(self, lat, lon):
+        self.lat = lat
+        self.lon = lon
+
 # Make a class Waypoint that can be passed parameters `name`, `lat`, and `lon` to the
 # constructor. It should inherit from LatLon. Look up the `super` method.
 
 # YOUR CODE HERE
 
+
+class Waypoint(Latlon):
+    def __init__(self, lat, lon, name):
+        super().__init__(lat, lon)
+        self.name = name
+
+    def __str__(self):
+        return f"""Name - {self.name} Latitude - {self.lat} Longitude - {self.lon}"""
+
+
 # Make a class Geocache that can be passed parameters `name`, `difficulty`,
 # `size`, `lat`, and `lon` to the constructor. What should it inherit from?
 
 # YOUR CODE HERE
+class Geocache(Waypoint):
+    def __init__(self, lat, lon, name, difficulty, size,):
+        super().__init__(lat, lon, name)
+        self.difficulty = difficulty
+        self.size = size
+
+    def __str__(self):
+        return f"""Name - {self.name} Latitude - {self.lat} Longitude - {self.lon} Difficulty - {self.difficulty} Size - {self.size} """
 
 # Make a new waypoint and print it out: "Catacombs", 41.70505, -121.51521
 
 # YOUR CODE HERE
 
+
+waypoint = Waypoint(41.70505, -121.51521, "Catacombs")
+
+
 # Without changing the following line, how can you make it print into something
 # more human-readable? Hint: Look up the `object.__str__` method
-print(waypoint)
+print(waypoint.name, waypoint.lat, waypoint.lon)
```

Comment:
Ideally this would have been done w/o changing the line (per the instructions). You defined the str method in Waypoint, which would have make this print correctly.